app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,12 @@
+class MerchantsController < ApplicationController 
+
+  def index 
+    @merchants = MerchantFacade.merchants
+  end
+
+  def show 
+    @merchant = MerchantFacade.merchant(params[:id]) 
+    @items = MerchantFacade.items(params[:id])
```

Comment:
Very minor comment, but I think this action would be more readable if the `MerchantFacade.items` was named with more specificity (something like `merchant_items`).